api/resource/origin.go

```diff
@@ -48,13 +55,6 @@ func (origin *Origin) Append(path string) *Origin {
 
 // String returns a string version of origin
 func (origin *Origin) String() string {
-	var anno string
-	anno = anno + "path: " + origin.Path + "\n"
-	if origin.Repo != "" {
-		anno = anno + "repo: " + origin.Repo + "\n"
-	}
-	if origin.Ref != "" {
-		anno = anno + "ref: " + origin.Ref + "\n"
-	}
-	return anno
+	anno, _ := kyaml.Marshal(origin)
```

Comment:
I think we always want to deal with return errors (e.g. checks like golangci lint shall fail if errors are siloed) 
I doubt if the rationale of the original code is to avoid handling the marshal return error.

Reply:
done, returning the error now